app/managers/performance_metrics_manager/index.test.ts

```diff
@@ -91,6 +98,26 @@ describe('load metrics', () => {
         expect(mockApiClient.post).toHaveBeenCalledWith(expectedUrl, expectedRequest);
     });
 
+    it('only register load once', async () => {
+        performance.mark('nativeLaunchStart');
+        const measure = getMeasure(TEST_EPOCH, 0);
+        const expectedRequest = getBaseReportRequest(measure.timestamp, measure.timestamp + 1);
+        expectedRequest.body.histograms = [measure];
+
+        PerformanceMetricsManager.setLoadTarget('HOME');
+        PerformanceMetricsManager.finishLoad('HOME', serverUrl);
+        await TestHelper.tick();
+        jest.advanceTimersByTime(INTERVAL_TIME);
+        await TestHelper.tick();
+        expect(mockApiClient.post).toHaveBeenCalledWith(expectedUrl, expectedRequest);
+        mockApiClient.post.mockClear();
+        PerformanceMetricsManager.finishLoad('HOME', serverUrl);
```

Comment:
Q: we expected to not see another call if we only call finishLoad() again, right?  Besides that scenario, is there any merit to see if we do another call to `performance.mark()` and then finishLoad() during the same "session"?

Reply:
Silly me, I misunderstood.  performance.mark() just instantiate a new class with value from `markName` and will not affect what you're testing here which is finishLoad().

Reply:
Follow-up suggestion, I think there might be a merit to cover the skipLoadMetric() just to make sure that no API calls are being posted.